joss_qmc/paper.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Quasi-Monte Carlo Methods in Python'
+tags:
+  - Python
+  - SciPy
+  - statistics
+  - Quasi-Monte Carlo methods
+authors:
+  - name: Pamphile T. Roy
+    affiliation: 1
+    corresponding: true
+    orcid: 0000-0001-9816-1416
+  - name: Art B. Owen
+    affiliation: 2
+    orcid: 0000-0001-5860-3945
+  - name: Matt Haberland
+    affiliation: 3
+    orcid: 0000-0003-4806-3601
+affiliations:
+    - name: Quansight
+      index: 1
+    - name: Stanford University
+      index: 2
+    - name: California Polytechnic State University, San Luis Obispo, USA
+      index: 3
+date: 30 August 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+
+
+# Statement of need
+
+NumPy random number generators (`numpy.random`) has become the standard way of
+sampling random numbers in the scientific Python ecosystem.
+These methods are fast and reliable, and the results are repeatable when a
+random seed is provided. However, sampling in high dimensions produces a lot of
+gaps and clusters of points. In integration problems, classical methods have a 
+low convergence rate meaning that a large sample size is required.
+
+By construction, Quasi-Monte Carlo (QMC) methods provide efficient, determinist
+(or not) and quality generators that can advantageously replace traditional
+methods. This can be decisive when the sampling size is limited or strong
+reproducibility guarantee is required. Also QMC methods are known to have
```

Comment:
This is true for the non scrambled versions. Because it's independent of any implementation or platform or seed as the sequence is exactly defined mathematically.

Reply:
Do you guarantee exact bitwise reproducibility? If so, I think we should mention that specifically. Otherwise, I don't see how `rng.default_rng(0)` is any less reproducible than `stats.qmc.Sobol(d, scramble=False)`.

Reply:
Yep it should be the case. For a given number of bits, the same algo will give exactly the same results on any platform (unscrambled again).

Reply:
Ok, then I'd be explicit about that, if that's an important selling point.

Reply:
I added a sentence for that.